fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationSwitchItem.kt

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.translations
+
+/**
+ * Translation switch list item that will appear on Translation screens.
+ */
+data class TranslationSwitchItem(
+    val isEnabled: Boolean,
+    val stateChange: (Boolean) -> Unit,
```

Comment:
```suggestion
    val onStateChange: (Boolean) -> Unit,
```
I assume this param is a callback.

Reply:
Done